active_contracts.json

```diff
@@ -60,11 +60,11 @@
     "PluginRepoRegistry": "0xc796bB1AfEBc56daDF6CAcD2aDa78055e5381971",
     "PluginRepoFactory": "0xDcC5933bc3567E7798Ff00Ab3413cF5f5801BD41",
     "PluginSetupProcessor": "0x9227b311C5cecB416707F1C8B7Ca1b52649AabEc",
-    "DAOFactory": "0x3ff1681f31f68Ff2723d25Cf839bA7500FE5d218",
+    "DAOFactory": "0x5bDBaAfd90B908058567080513635f560F896918",
     "AddresslistVotingSetup": "0xD1CE76c26bc20255157C8ef478276eb115eC02a7",
     "TokenVotingSetup": "0x76D3795Df5acF2e2141557BA1Cf094fd54d4e10C",
     "AdminSetup": "0x40a3EF0f0780e044EbDDEdAa9AB225158f315afd",
-    "MultisigSetup": "0x2C233F09CaB08BC66b750CB80da401Bd700B74a0",
+    "MultisigSetup": "0x8b2361C65E898F40372a0DD5E1BfEE9895A676B3",
```

Comment:
I am trying to understand why the two contracts changed on mumbai?

Reply:
we already "made" a partial update on Mumbai. Meaning we deployed the contracts but nobody made the proposal in the managing DAO.
When I did this I assumed that we will apply this update so I already updated the active_contracts.json. Now we know that this update never happened so I reverted the addresses to be equal to the 1.0.0 deployment on Mumbai so we properly revert the permissions on the DAOFactory

Reply:
Ok, thanks for clarifying